openvdb_cmd/vdb_tool/include/Geometry.h

```diff
@@ -0,0 +1,1126 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+////////////////////////////////////////////////////////////////////////////////
+///
+/// @author Ken Museth
+///
+/// @file Geometry.h
+///
+/// @brief Class that encapsulates (explicit) geometry, i.e. vertices/points,
+///        triangles and quads. It is used to represent points and polygon meshes
+///
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef VDB_TOOL_GEOMETRY_HAS_BEEN_INCLUDED
+#define VDB_TOOL_GEOMETRY_HAS_BEEN_INCLUDED
+
+#include <functional>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <cstdlib> // for std::malloc and std::free
+
+#include <tbb/parallel_for.h>
+#include <tbb/parallel_reduce.h>
+#include <tbb/blocked_range.h>
+
+#include <openvdb/openvdb.h>
+#include <openvdb/points/PointCount.h>
+
+#ifdef VDB_TOOL_USE_NANO
+#include <nanovdb/NanoVDB.h>
+#include <nanovdb/util/IO.h>
+#endif
+
+#ifdef VDB_TOOL_USE_ABC
+#include <Alembic/Abc/TypedPropertyTraits.h>
+#include <Alembic/AbcCoreAbstract/All.h>
+#include <Alembic/AbcCoreFactory/All.h>
+#include <Alembic/AbcCoreOgawa/All.h>
+#include <Alembic/AbcGeom/All.h>
+#include <Alembic/Util/All.h>
+#endif
+
+#if defined(_WIN32)
+#include <io.h>
+#else
+#include <unistd.h>
+#endif
+
+#include "Util.h"
+
+namespace openvdb {
+OPENVDB_USE_VERSION_NAMESPACE
+namespace OPENVDB_VERSION_NAME {
+namespace vdb_tool {
+
+class Geometry
+{
```

Comment:
As a general comment, the member variables are usually included at the end of the class definition.